### PR TITLE
a little stuck

### PR DESCRIPTION
**Pull request by dechmann**

### CODE REVIEW

**Comment by dechmann** [comment]
done

File: favorites.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (mbMosman) [inline favorites.json]: JSON well formatted and includes list :heavy_check_mark:

File: js/favorites.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (mbMosman) [inline js/favorites.json]: JSON well formatted and includes list :heavy_check_mark:

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+$(function() {
+    
+    
+    
+    var $body = $("body");
+    //removes the put code here
+    $("p").addClass("hide").remove();
+    
+    //This will display my json to the html.
+    $.getJSON("js/favorites.json")
```
Comment (mbMosman) [inline js/script.js]: Ajax includes error handling

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    .done( function(data){
+        for(i = 0; i < data.favorite.length; i++){
+            //$body.append("<H2>" + data.favorite[i].name + "</H2>");
+            //$body.append("<p>" + data.favorite[i].info + "</p>");
+            //$body.append("<ul>" + createList(data.favorite[i].list) + "</ul>");
+            $body.append("<div><h2>" + data.favorite[i].name + "</h2><p>" + data.favorite[i].info + "</p><ul>" + createList(data.favorite[i].list) + "</ul>");
```
Comment (mbMosman) [inline js/script.js]: You could put the building of this HTML into a function that returned back a string.  Might make it a little easier to read, kinda like you did for the part that builds the list items.

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+$(function() {
+    
+    
+    
+    var $body = $("body");
+    //removes the put code here
+    $("p").addClass("hide").remove();
```
Comment (mbMosman) [inline js/script.js]: no need to add a hide class if you're just going to remove it.

File: js/script.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
+$(function() {
```
Comment (mbMosman) [inline js/script.js]: Code inside jQuery ready :heavy_check_mark:

File: js/script.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+            //$body.append("<p>" + data.favorite[i].info + "</p>");
+            //$body.append("<ul>" + createList(data.favorite[i].list) + "</ul>");
+            $body.append("<div><h2>" + data.favorite[i].name + "</h2><p>" + data.favorite[i].info + "</p><ul>" + createList(data.favorite[i].list) + "</ul>");
+        }
+        //this is to hide the content and show content.
+        $("p").hide();
+        $("ul").hide();
+        
+        //this is my click handeler.
+        $("h2").on("click", function() {
```
Comment (mbMosman) [inline js/script.js]: jQuery on function used for event handling. :heavy_check_mark:

**Comment by mbMosman** [comment]
Nice work!

**Pull request closed without merging**